libraries/stdlib/samples/test/samples/collections/maps.kt

```diff
@@ -353,6 +353,11 @@ class Maps {
             assertPrints(peopleToAge.map { it.value }, "[20, 21]")
         }
 
+        @Sample
+        fun flatMap() {
+            val map = mapOf("122" to 2, "3455" to 3)
+            assertPrints(map.flatMap { entry -> entry.key.take(entry.value).toList() }, "[1, 2, 3, 4, 5]")
```

Comment:
It would be more succinct to destructure the entry in lambda parameters: `flatMap { (key, value) -> ... }`

Reply:
done